lib/Braintree/Digest.php

```diff
@@ -50,8 +50,8 @@ public static function _hmacSha1($message, $key)
         $outerPad = str_repeat(chr(0x5C), 64);
 
         for ($i = 0; $i < 20; $i++) {
-            $innerPad{$i} = $keyDigest{$i} ^ $innerPad{$i};
-            $outerPad{$i} = $keyDigest{$i} ^ $outerPad{$i};
+            $innerPad[$i] = $keyDigest[$i] ^ $innerPad[$i];
```

Comment:
I assume this syntax works in php 7.2?

Reply:
https://wiki.php.net/rfc/deprecate_curly_braces_array_access

Up until now both syntaxes have been interchangeable and working at least since PHP5

This is a demo of both syntaxes being used on PHP5.6
https://www.tehplayground.com/8V5WHa6O3bqXLAXp